downstream/modules/eda/proc-modifying-activations-during-install.adoc

```diff
@@ -0,0 +1,14 @@
+[id="modifying-activations-during-install"]
+
+= Modifying the number of simultaneous rulebook activations during {EDAcontroller} installation 
+
+[role="_abstract"]
+By default, {EDAcontroller} allows 12 activations to run simultaneously. You can modify this default value during installation by using the following procedure:
+
+.Procedure
+Provide a variable to the VM installer:
+
+. Navigate to the setup inventory file. 
+. Add `automationedacontroller_max_running_activations` under [all:vars] section.
+
```

Comment:
Lets given an example here: 

```
automationedacontroller_max_running_activations=16
```

Reply:
Hi @jainnikhil30! Same as the last comment, could you take a look at the formatting of this example? Thanks!